packages/venia-ui/lib/components/SignIn/__tests__/signIn.spec.js

```diff
@@ -165,30 +165,3 @@ test('changes view to ForgotPassword', () => {
     expect(setDefaultUsername).toHaveBeenCalledTimes(1);
     expect(showForgotPassword).toHaveBeenCalledTimes(1);
 });
-
-test("avoids reading the form if it doesn't exist", () => {
```

Comment:
This also seems like something we don't need to actually test. I removed it, but I could be convinced to add it back if someone feels strongly about it (and has a way to actually make it work).